peripherals/opamp/opamp_g4.yaml

```diff
@@ -0,0 +1,161 @@
+
+OPAMP:
+  OPAMP?_CSR:
+    LOCK:
+      ReadWrite: [0, "CSR is read-write"]
+      ReadOnly: [1, "CSR is read-only, can only be cleared by system reset"]
+    CALOUT: [0, 1]
+    TRIMOFFSETN: [0, 31] 
+    TRIMOFFSETP: [0, 31]
+    PGA_GAIN:
+      Gain2: [0, "Gain 2"]
+      Gain4: [1, "Gain 4"]
+      Gain8: [2, "Gain 8"]
+      Gain16: [3, "Gain 16"]
+      Gain32: [4, "Gain 32"]
+      Gain64: [5, "Gain 64"]
+      Gain2_InputVINM0: [8, "Gain 2, input/bias connected to VINM0 or inverting gain"]
+      Gain4_InputVINM0: [9, "Gain 4, input/bias connected to VINM0 or inverting gain"]
+      Gain8_InputVINM0: [10, "Gain 8, input/bias connected to VINM0 or inverting gain"]
+      Gain16_InputVINM0: [11, "Gain 16, input/bias connected to VINM0 or inverting gain"]
+      Gain32_InputVINM0: [12, "Gain 32, input/bias connected to VINM0 or inverting gain"]
+      Gain64_InputVINM0: [13, "Gain 64, input/bias connected to VINM0 or inverting gain"]
+      Gain2_FilteringVINM0: [16, "Gain 2, with filtering on VINM0"]
+      Gain4_FilteringVINM0: [17, "Gain 4, with filtering on VINM0"]
+      Gain8_FilteringVINM0: [18, "Gain 8, with filtering on VINM0"]
+      Gain16_FilteringVINM0: [19, "Gain 16, with filtering on VINM0"]
+      Gain32_FilteringVINM0: [20, "Gain 32, with filtering on VINM0"]
+      Gain64_FilteringVINM0: [21, "Gain 64, with filtering on VINM0"]
+      Gain2_InputVINM0FilteringVINM1: [24, "Gain 2, input/bias connected to VINM0 with filtering on VINM1 or inverting gain"]
```

Comment:
These enums could be customized to Describe actual Pins where VINM0 VINM1 are connected. But IMO this would make this yml bit too big, especially if all 6 opamps were to be implemented